pipeline/outputs/s3.md

```diff
@@ -26,6 +26,7 @@ See [here](https://github.com/fluent/fluent-bit-docs/tree/43c4fe134611da471e706b
 | total_file_size               | Specifies the size of files in S3. Maximum size is 50G, minimim is 1M.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 | 100M                                    |
 | upload_chunk_size             | The size of each 'part' for multipart uploads. Max: 50M                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                | 5,242,880 bytes                         |
+| MAX_FILE_SIZE_PUT_OBJECT             | Max file size allowed to publish to S3 MAX: 1GB.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          | 1,073,741,824 bytes                         |
```

Comment:
this isn't a new user config option. Its a different max size for `total_file_size` for the use_put_object case